packages/file-search/src/node/file-search-service-impl.spec.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import 'mocha';
+import * as chai from 'chai';
+import * as path from 'path';
+import { FileSearchServiceImpl } from './file-search-service-impl';
+import { FileUri } from '@theia/core/lib/node';
+import { MockLogger } from '@theia/core/lib/common/test/mock-logger';
+
+const expect = chai.expect;
+
+describe('search-service', () => {
+    it('shall fuzzy search this spec file', async () => {
+        const service = new FileSearchServiceImpl(new MockLogger());
+        const uri = FileUri.create(path.resolve(__dirname, "../../../.."));
+        const matches = await service.find(uri.toString(), 'spc');
+        const testFile = matches.find(e => e.endsWith(__filename));
+        expect(testFile).to.eql(FileUri.create(__filename).toString());
```

Comment:
I wanted to have a bigger workspace, to also get an idea of timing. In case we move it out, I will be happy to fix this again.

Reply:
I don't think this is acceptable because:

- We're knowing pushing work downstream and we should avoid that and we can avoid it.
- When we move out you will lose the big repo aspect and have to redo this extensively.
- Other extensions are likely to need something like this thus we're losing the opportunity to reuse good testing code.

Reply:
fixed it.